tests/test_run_ab_transforms.py

```diff
@@ -50,6 +56,32 @@ def test_run_ab_transforms_success(
         )
 
 
+def test_run_ab_transforms_raise_error(
```

Comment:
I'd be clearer about what is triggering the error in the test name

Reply:
Renamed test to `test_run_ab_transforms_raise_error_if_containers_failed`.